### PR TITLE
Flashbang sound stun nerf

### PR DESCRIPTION
**Pull request by TheShown911**
## About The Pull Request

Flashbangs have the `weaken` on the bang proc connected to ear protection removed and the `stun` reduced by 50%. `weaken` and `stun` on the check related to eye protection remain the same. 
## Why It's Good For The Game

Ear protection is almost non-existant in the game, and even *with* ear protection people could stll get knocked down.

## Changelog
:cl: TheShown
balance: Flashbangs no longer knockdown people with eye protection, and the stun on people with eye protection has been further reduced.
/:cl:

### CODE REVIEW

**Review by TorinoFermic — APPROVED** [review]

**Review by genessee596 — APPROVED** [review]
Delegate stamp of approval.

Flashbang spam is a shit tactic that really shouldn't work as well as it does.

Has maintainer approval and delegate approval, so I'll merge this now.

**Pull request merged**
